test/testinput/bufr_ncep_mtiasi.yaml

```diff
@@ -0,0 +1,206 @@
+          # MetaData
+          timestamp:
+            datetime:
+              year: "*/YEAR"
+              month: "*/MNTH"
+              day: "*/DAYS"
+              hour: "*/HOUR"
+              minute: "*/MINU"
+
+          latitude:
+            query: "*/CLATH"
+
+          longitude:
+            query: "*/CLONH"
+
+          satelliteId:
+            query: "*/SAID"
+
+          sensorId:
+            query: "*/SIID[1]"
+
+          scanLineNumber:
+            query: "*/SLNM"
+
+          gqisFlagQual:
+            query: "*/QGFQ"
+
+          fieldOfViewNumber:
+            query: "*/FOVN"
+
+          solarZenithAngle:
+            query: "*/SOZA"
+
+          solarAzimuthAngle:
+            query: "*/SOLAZI"
+
+          sensorZenithAngle:
+            query: "*/SAZA"
+
+          sensorAzimuthAngle:
+            query: "*/BEARAZ"
+
+          heightOfStation:
+            query: "*/SELV"
+
+          startChannel:
+            query: "*/IASIL1CB/STCH"
+
+          endChannel:
+            query: "*/IASIL1CB/ENCH"
+
+          channelScaleFactor:
+            query: "*/IASIL1CB/CHSF"
+
+          sensorChannelNumber:
+            query: "*/IASICHN/CHNM"
+
+          # ObsValue
+          scaledSpectralRadiance:
+            query: "*/IASICHN/SCRA"
+
+          fractionOfClearPixelsInFov:
+            query: "*/IASIL1CS/FCPH"
+            transforms:
+               - scale: 0.01
+
+        splits:
+          satId:
+            category:
+              variable: satelliteId
+              map:
+                _3: metop-b
+                _4: metop-a
+                _5: metop-c
+
+    ioda:
+      backend: netcdf
+      obsdataout: "./testrun/gdas.t12z.mtiasi.{splits/satId}.tm00.nc"
+
+      dimensions:
+        - name: Channel
+          path: "*/IASICHN"
+        - name: Cluster 
+          path: "*/IASIL1CS"
+        - name: Band 
+          path: "*/IASIL1CB"
+
+      globals:
+        - name: "platformCommonName"
+          type: string
+          value: "IASI"
+
+        - name: "platformLongDescription"
+          type: string
+          value: "MTYP 021-027 PROCESSED MHS Tb"
+
+      variables:
+
+        # MetaData 
+        - name: "MetaData/dateTime"
+          source: variables/timestamp
+          longName: "Datetime"
+          units: "seconds since 1970-01-01T00:00:00Z"
+
+        - name: "MetaData/latitude"
+          source: variables/latitude
+          longName: "Latitude"
+          units: "degrees_north"
+          range: [ -90, 90 ]
+
+        - name: "MetaData/longitude"
+          source: variables/longitude
+          longName: "Longitude"
+          type: float 
+          units: "degrees_east"
+          range: [ -180, 180 ]
+
+        - name: "MetaData/satelliteIdentifier"
+          source: variables/satelliteId
+          longName: "Satellite Identifier"
+
+        - name: "MetaData/satelliteInstrument"
+          source: variables/sensorId
+          longName: "Satellite Instrument"
+
```

Comment:
instrumentIdentifier, see convention table.  Double check if you need to correct ??